README.md

```diff
@@ -235,6 +235,22 @@ import chartmogul
 chartmogul.Transaction.create(config, uuid='inv_745df1d4-819f-48ee-873d-b5204801e021', data={})
 ```
 
+#### [SubscriptionEvents](https://dev.chartmogul.com/docs/subscription_events)
+
+```python
+import chartmogul
+chartmogul.SubscriptionEvent.all(config)
+chartmogul.SubscriptionEvent.create(config, data={})
```

Comment:
Can we change this to actually parametrized version? Because currently this won't work since you cannot create empty subscription event, right?